### DISCUSSION TITLE
Bug: single-letter subdomains not parsing in output

### DISCUSSION THREAD

**Discussion by travisdowns** [community/community#25732] [Actions]
<p>Github actions fails to parse single-letter subdomains (and single-letter domains, but that’s probably a much less common problem).</p>
<p>When you write a URL to standard out/err in a GitHub actions task, the UI parses it as a URL and makes the link clickable.</p>
<p>However, if the URL has a single letter subdomain, the parsing fails.</p>
<p>Here is an <a href="https://github.com/travisdowns/gha-test/runs/3883571690?check_suite_focus=true" rel="noopener nofollow ugc">example run showing the problem</a>. Open URL test and you’ll see that <code>https://a.example.com</code> fails to parse, along with <code>a.co</code>, but the URLs without single letter domains parse fine:</p>
<p><img src="https://user-images.githubusercontent.com/2403521/181092927-f7cb411e-74e0-4ae0-b1a3-3b1fd16e1055.png" alt="image" data-base62-sha1="cW1h823S4pc31yWLOz2iHG4MBHa" width="376" height="260"></p>
<p>Single-letter subdomains are totally valid and should be parsed correctly. I created one for the express purpose of shortening URLs printed in GitHub actions, so this behavior bit me recently.</p>

**Comment by Zerotask** [comment]
<p>You can report this bug to GitHub directly: <a href="https://support.github.com/contact/feedback" class="inline-onebox" rel="noopener nofollow ugc">Share feedback - GitHub Support</a></p>

**Comment by travisdowns** [comment]
<p>Thanks, I’ve submitted it there as well.</p>

**Comment by konradpabjan** [comment, marked as answer]
<p><a class="mention" href="/travisdowns">@travisdowns</a> Thanks for submitting the feedback</p>
<p>We’ve deployed a change today that should correctly highlight single-letter subdomains in logs</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/c/5/c5c96348c96b6c7a2a5a1c60b322cbe9e17ee6d7.png" data-download-href="/uploads/short-url/sdHx3hKRhdd3uQ9AnD3LGcaGAHd.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/16109154/181092930-c3bcd7d5-9999-4645-8c4e-9fd548d8da32.png" alt="image" data-base62-sha1="sdHx3hKRhdd3uQ9AnD3LGcaGAHd" width="690" height="218" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c5c96348c96b6c7a2a5a1c60b322cbe9e17ee6d7_2_690x218.png, https://aws1.discourse-cdn.com/github/original/3X/c/5/c5c96348c96b6c7a2a5a1c60b322cbe9e17ee6d7.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/c/5/c5c96348c96b6c7a2a5a1c60b322cbe9e17ee6d7.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/c/5/c5c96348c96b6c7a2a5a1c60b322cbe9e17ee6d7_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">850×269 21.8 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by travisdowns** [comment]
<p>Thanks, it works!</p>
<p>Nice turnaround time, much appreciated.</p>
